### PR TITLE
[Sylius/Behat] Use getUrl() instead of custom open() in ProductShowPage

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

File: src/Sylius/Behat/Page/Product/ProductShowPage.php [hunk 1]
```diff
@@ -61,7 +48,15 @@ public function addToCartWithVariant($variant)
...
+    protected function getRouteName()
+    {
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getUrl(array $urlParameters = [])
     {
+        return $this->router->generate($urlParameters['product'], [], true);
```
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Page/Product/ProductShowPage.php]: Maybe we should add check if `product` is set?
```
if (!isset($urlParameters['product']) {
  \\ throw exception
}
```

Comment (pamil) [inline src/Sylius/Behat/Page/Product/ProductShowPage.php]: :+1:

**Comment by pamil** [comment]
Something went wrong with Travis, here's the passing build on my fork: https://travis-ci.org/pamil/Sylius/jobs/108260994

**Comment by michalmarcinkowski** [comment]
@pamil the passing build from your fork is not enough :smile: please repush, should be fine then.

**Comment by michalmarcinkowski** [comment]
:+1:

**Pull request merged**